### PR TITLE
Updated release versions for compose and helm

### PR DESCRIPTION
**Pull request by sujaya-sys**
Thank you for submitting this pull request! We appreciate you spending the time to work on these changes.

Updated release versions for compose and helm

## Have you read the [Contributing Guidelines](CONTRIBUTING.md)?

- [x] I have read the [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: content/en/docs/get started/included/install-score-compose-curl-mac.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ toc_hide: true
 ### Step 1. Download the latest release
 
 ```bash
-curl -L0 "https://github.com/score-spec/score-compose/releases/download/0.2.0/score-compose_0.2.0_darwin_arm64.tar.gz" -o score-compose.tgz
+curl -L0 "https://github.com/score-spec/score-compose/releases/download/0.2.0/score-compose_0.12.2_darwin_arm64.tar.gz" -o score-compose.tgz
```
Comment (TobiasBabin) [inline content/en/docs/get started/included/install-score-compose-curl-mac.md:11]:
```suggestion
curl -L0 "https://github.com/score-spec/score-compose/releases/download/0.12.2/score-compose_0.12.2_darwin_arm64.tar.gz" -o score-compose.tgz
```

File: content/en/docs/get started/included/install-score-helm-curl-mac.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ toc_hide: true
 ### Step 1. Download the latest release
 
 ```bash
-curl -L0 "https://github.com/score-spec/score-helm/releases/download/0.2.0/score-helm_0.2.0_darwin_arm64.tar.gz" -o score-helm.tgz
+curl -L0 "https://github.com/score-spec/score-helm/releases/download/0.2.0/score-helm_0.9.0_darwin_arm64.tar.gz" -o score-helm.tgz
```
Comment (TobiasBabin) [inline content/en/docs/get started/included/install-score-helm-curl-mac.md:11]:
```suggestion
curl -L0 "https://github.com/score-spec/score-helm/releases/download/0.9.0/score-helm_0.9.0_darwin_arm64.tar.gz" -o score-helm.tgz
```

**Review by TobiasBabin — CHANGES_REQUESTED** [review]
The download links need the version number updated in two places each. Please check the commit suggestions.

**Comment by sujaya-sys** [comment]
Totally missed that 🤦‍♀️  Thanks Tobi!

**Pull request merged**